colorbleed/plugins/maya/publish/validate_rendersettings.py

```diff
@@ -15,12 +15,18 @@ class ValidateRenderSettings(pyblish.api.InstancePlugin):
 
     * Frame Padding must be:
         * default: 4
+
+    NOTE:
+        The repair function of this plugin will does not repair animation
```

Comment:
What are these "multiple possibilities"?

Also, typo: ` will does not` -> ` will not`  or `does not`.

Reply:
> What are these "multiple possibilities"?

You have `Standard` and `Specific Frames` which both return `True` when querying the animation attribute of `defaultGlobals`. The difference lies in how to query the frame range. I have not dived in to this further.

Reply:
Ah yes, for now we are not even supporting Specific Frames, correct?

Specific Frames can set e.g. 1, 3, 7, 10 (to just render those). The syntax also supports ranges. Pretty dope. Actually, thinking about it that's an issue on its own (for YouTrack?). Rarely, if ever, used by the renderteam. ;) Might also conflict with how we publish sequences currently - we only allow consecutive sequences.

Reply:
We will need to see how this effects the other renders. I noticed that Arnold for example uses the file name template from a dropdown menu to check if it can render animation. It needs to be set to `<fname>.#.<ext>` or something similar.

I do not know how Redshift goes about its way to recognize a sequence of frames. Will take a look at that now.

Reply:
Relieve! Redshift uses a simple checkbox for this which has the same name.